TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -312,10 +312,14 @@ TYPO3:
           'TYPO3.Neos/Inspector/Editors/ReferencesEditor':
             editorOptions:
               placeholder: 'TYPO3.Neos:Main:typeToSearch'
+              nodeTypes: ['Neos.Neos:Document']
```

Comment:
Thanks to @aertmann for fixing this in https://github.com/neos/neos-development-collection/commit/3fb088cac26fe649158ae4166eb8f185975481ce